js/digifinex.js

```diff
@@ -0,0 +1,528 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { AccountSuspended, BadRequest, BadResponse, NetworkError, DDoSProtection, AuthenticationError, PermissionDenied, ArgumentsRequired, ExchangeError, InsufficientFunds, InvalidOrder, InvalidNonce } = require ('./base/errors');
+const { DECIMAL_PLACES } = require ('./base/functions/number');
+// const log = require ('ololog');
```

Comment:
Wanted to ask you to also cleanup these commented debug logs throughout this file, pls )